openmicroscopy/bioformats
# Overlapped tiling example and docs

This PR contains a number of updates to the tiling example files and documentation.

- Includes an example file (OverlappedTiledWriter) for tiled reading and writing in scenarios ()
- Fixes a bug in the existing TiledReaderWriter which was not properly initialising the tiling parameters
- Removes the catching of exceptions from the existing examples to allow for the above bug to be caught by CI in case of regression
- Adds an example of using the sizeX and sizeY parameters to the fake files in ExampleSuite
- Updates the tiling sphinx documentation to include further description of overlapped tiling examples

Testing required:
- Ensure that the ExampleSuite runs successfully with the new OverlappedTiledWriter example included
- Ensure all existing examples in that ExampleSuite continue to pass
- Check all the links and formatting in the updated sphinx doc and proof read
